.github/workflows/continuous-integration-workflow.yml

```
@@ -11,21 +11,21 @@ on:
 
 jobs:
   CI:
-    continue-on-error: ${{ matrix.config.continue-on-error == 'true' }}
     strategy:
-      matrix:
-        config:
-          - {dockerfile: 'fedora',   tag: 'latest'}
-          - {dockerfile: 'fedora',   tag: 'rawhide',          build_args: 'TAG=rawhide', continue-on-error: 'true'}
-          - {dockerfile: 'ubuntu',   tag: 'latest'}
-          - {dockerfile: 'ubuntu',   tag: 'rolling',          build_args: 'TAG=rolling'}
-          - {dockerfile: 'ubuntu',   tag: 'devel',            build_args: 'TAG=devel', continue-on-error: 'true'}
-          - {dockerfile: 'opensuse', tag: 'latest'}
-          - {dockerfile: 'fedora',   tag: 'intel',            build_args: 'TAG=38,INTEL=yes', continue-on-error: 'true'}
+        matrix:
+          config:
+            - {dockerfile: 'fedora',   tag: 'latest'}
+            - {dockerfile: 'fedora',   tag: 'rawhide',  build_args: 'TAG=rawhide',  continue-on-error: 'true'}
+            - {dockerfile: 'ubuntu',   tag: 'latest'}
+            - {dockerfile: 'ubuntu',   tag: 'rolling',  build_args: 'TAG=rolling'}
+            - {dockerfile: 'ubuntu',   tag: 'devel',    build_args: 'TAG=devel',    continue-on-error: 'true'}
+            - {dockerfile: 'opensuse', tag: 'latest'}
+            - {dockerfile: 'fedora',   tag: 'intel',    build_args: 'TAG=38,INTEL=yes',    cmake_args: '-DCMAKE_CXX_COMPILER=icpc',  continue-on-error: 'true'}
```

This needs to be
```suggestion
            - {dockerfile: 'fedora',   tag: 'intel',    build_args: 'TAG=37,INTEL=yes',    cmake_args: '-DCMAKE_CXX_COMPILER=icpc',  continue-on-error: 'true'}
```
see #45 and #46.